features/support/hooks.rb

```diff
@@ -0,0 +1,3 @@
+Before("@star")do
```

Comment:
Here I chosed to use a before hook in place of a background, because I've read that background can be used for describing actions useful for the customer, while hooks are best used for actions more technically involved. I think that maybe creating a snippet before testing somehow blurs the feature.